public/images/favicon/browserconfig.xml

```diff
@@ -2,9 +2,11 @@
 <browserconfig>
     <msapplication>
         <tile>
-            <square150x150logo src="/images/favicon/mstile-150x150.png?v=fi5jKKtbwv"/>
-            <square310x310logo src="/images/favicon/mstile-310x310.png?v=fi5jKKtbwv"/>
-            <TileColor>#2973ff</TileColor>
+            <square70x70logo src="/images/favicon/mstile-70x70.png?v=yHh3NA"/>
+            <square150x150logo src="/images/favicon/mstile-150x150.png?v=yHh3NA"/>
+            <square310x310logo src="/images/favicon/mstile-310x310.png?v=yHh3NA"/>
+            <wide310x150logo src="/images/favicon/mstile-310x150.png?v=yHh3NA"/>
```

Comment:
this is missing.

Reply:
agh, I need to stop using `commit -a`. Fixed